### ISSUE TITLE
Rsync test fails (parsing error)

### ISSUE THREAD

**Issue by zeechs** [time2backup/time2backup#212]
Here is why. This is the output of my rsync command:
```
...

Number of files: 47 (reg: 27, dir: 20)
Number of created files: 47 (reg: 27, dir: 20)
Number of deleted files: 0
Number of regular files transferred: 27
Total file size: 25.003 bytes
Total transferred file size: 25.003 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 1.440
File list generation time: 0,001 seconds
File list transfer time: 0,000 seconds
Total bytes sent: 449
Total bytes received: 1.612

sent 449 bytes  received 1.612 bytes  588,86 bytes/sec
total size is 25.003  speedup is 12,13 (DRY RUN)
```

The interesting line is the following:
`Total transferred file size: 25.003 bytes`

Notice that the thousand separator is a dot. This line is parsed [here](https://github.com/time2backup/time2backup/blob/master/inc/functions.sh#L2154) and is expected to use commas, as thousand separator. The function fails and t2b does not perform the backup.

On my system, the `LANG` env variable is set to `fr_FR.utf-8`. Setting `LANG=C` fixes the problem. Other locales are probably affected by this issue.

I would submit a PR but I'm not sure how you would like to fix this.
- Override the LANG env variable to get consistent rsync output across all systems
- Fix the sed command to handle all possible thousand separators
- ...

**Comment by pruje** [comment]
Thanks for report this.
It seems there are only 2 separators that exists in the world: dots and commas.
So I would recommend to handle this inside the sed syntax.

Feel free to open a PR. Thanks a lot!

**Comment by zeechs** [comment]
Sorry, but I will have to let you handle the fix.
I didn't use github for a while and I just found out that I can't use my password for pushing commits to a repo anymore. I would be happy to help, but I don't have time nor patience for microsoft bs.
Anyway, thank you for this great util :+1:

**Comment by pruje** [comment]
Bug fixed in commit c9136b34542ddd78e3758d88436c55af5a4527d1 and published in 1.9.1 that was released tonight
